core/sql/optimizer/RelExeUtil.cpp

```diff
@@ -3733,6 +3734,23 @@ RelExpr * DDLExpr::bindNode(BindWA *bindWA)
         return NULL;
       }
 
+      // Hive tables can only be specified as external and must be created
+      // with the FOR clause
+      if (qualObjName_.isHive()) 
+      {
+        if (createTableNode->isExternal())
+        {
+          externalTable = TRUE;
+          isHbase_ = TRUE;
```

Comment:
Just checking: We set isHbase_ to TRUE for Hive tables? Is there an isHive_ member in the class?